commit 5b8b6c5933599642e2be322bdc3e081e7c981c3c (refs/changes/05/2305/2)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-07-31T12:44:26-07:00 (2 years, 2 months ago)
    
    DOCKER-1081 test api-images-v1-v2.test.js failing due to wrong imgapi image layer
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/test/integration/api-images-v1-v2.test.js b/test/integration/api-images-v1-v2.test.js
index dd4e527..280ac89 100644
--- a/test/integration/api-images-v1-v2.test.js
+++ b/test/integration/api-images-v1-v2.test.js
@@ -230,7 +230,7 @@ test('init docker images', function (tt) {
                     + contentLayers.length));
                 return;
             }
-            var digestChain = manifest.layers.slice(0, layerIdx).map(
+            var digestChain = manifest.layers.slice(0, layerIdx+1).map(
                 function (layer) {
                     return layer.digest;
                 }
